solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/filter/SetAccessStatusFilterTest.java

```diff
@@ -17,6 +17,7 @@
 
 import static edu.unc.lib.dl.acl.util.AccessPrincipalConstants.AUTHENTICATED_PRINC;
 import static edu.unc.lib.dl.acl.util.AccessPrincipalConstants.PUBLIC_PRINC;
+import static java.util.Arrays.asList;
```

Comment:
Will probably want to add a test or two to verify that collections get the right patron-settings tag